database-plugins/src/main/java/io/cdap/plugin/db/common/FQNGenerator.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.db.common;
+
+import io.opentracing.contrib.jdbc.ConnectionInfo;
+import io.opentracing.contrib.jdbc.parser.URLParser;
+
+/**
+ * Generate FQN from DB URL Connection
+ */
+
+public class FQNGenerator {
+  private static final String POSTGRES_DEFAULT_SCHEMA = "public";
+
+  public static String constructFQN(String url, String tableName) {
+    ConnectionInfo connectionInfo = URLParser.parse(url);
+    // DB type as set by library after extraction
+    if (connectionInfo.getDbType() == "postgresql") {
```

Comment:
qq - `Database schema` is also supported by  `oracle` and `sql server` but we are using it in generating FQN for only `postgres`. Is this intentional??

Reply:
yes, this is intentional - https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md

Reply:
I don't see any information about oracle and sql server in this documentation which you pointed above, why can't we have a similar naming convention for all the databases which support schema??

Reply:
discussed offline.